plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -294,6 +294,12 @@ public function migrate( $args = array(), $assoc_args = array() ) {
 	 * default: -1
 	 * ---
 	 *
+	 * [--verbose=<verbose>]
```

Comment:
I think it's a common that thing is a flag without any value, therefore invoking `command --verbose` implies output verbosity.

Reply:
Seems like WP CLI supports this out of the box, so we can run the command with and without `--verbose` and it will have the expected output.